components/gitpod-db/src/typeorm/entity/db-team-settings.ts

```diff
@@ -25,34 +25,34 @@ export class DBOrgSettings implements OrganizationSettings {
     workspaceSharingDisabled?: boolean;
 
     @Column("varchar", { nullable: true })
-    defaultWorkspaceImage?: string | null;
+    defaultWorkspaceImage?: string;
```

Comment:
The `null` is treated as special value by typeorm to clear the column.
This is the only place we did this, and it's also unnecessary: We treat the default value `""` the same anyway. :shrug: So we can get rid of that complexity.